src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/TimeoutsTest.java

```diff
@@ -170,16 +180,19 @@ public void cleanResources() {
     @Test
     public void testTimeoutsConfigured() throws Exception {
         final DatabaseEngine de = DatabaseFactory.getConnection(this.dbProps);
+        final Connection connection = de.getConnection();
 
-        int loginTimeoutInSeconds = DriverManager.getLoginTimeout();
-        int socketConnectionTimeoutInMs = de.getConnection().getNetworkTimeout();
-        int checkConnectionTimeoutInSec = de.getProperties().getCheckConnectionTimeout();
```

Comment:
I know these are kind of useless, but I'd rather keep them.
I think they make it more clear that some timeouts are seconds and others ms,
instead of having to infer those time units by looking at the expected constants in the assertions.